bentoml/README.md

```diff
@@ -0,0 +1,30 @@
+# vulnerble version setup
+
+```
+apt install python3.10 python3.10-venv python3-pip 
+python3 -m venv . && source bin/activate
+pip install torch transformers bentoml==1.2.0
+bentoml serve service:Summarization
+```
+run the exploit:
+```
+ncat -klnv 1337
+python3 exploit.py
+```
+you'll receive a http request on port 1337 which means the exploit worked.
+
+# safe version setup
+
+```
+apt install python3.10 python3.10-venv python3-pip 
+python3 -m venv . && source bin/activate
+pip install torch transformers bentoml==1.2.5
+bentoml serve service:Summarization
+```
```

Comment:
Can you please turn these in two docker compose files or similar?

I already had to do that to test the plugin, so you're free to copy-paste these (or create your own if you prefer):

`Dockerfile`
```Dockerfile
FROM python:3.10-alpine
RUN mkdir /workdir
WORKDIR /workdir
COPY service.py /workdir/
RUN apk add gcc python3-dev musl-dev linux-headers curl
ARG BENTOML_VERSION
RUN pip3 install bentoml==${BENTOML_VERSION}
CMD bentoml serve service:Summarization
```

`docker-compose-vulnerable.yml`
```yml
services:
  bentoml-vuln:
    build:
      context: .
      args:
        BENTOML_VERSION: 1.2.0
    ports:
      - "3000:3000"
```

`docker-compose-safe.yml`
```yml
services:
  bentoml-safe:
    build:
      context: .
      args:
        BENTOML_VERSION: 1.2.5
    ports:
      - "3000:3000"
```